src/compile/data/impute.ts

```diff
@@ -1,8 +1,11 @@
+import {normalizeimputeProperties} from '../../impute';
 import {ImputeSequence, ImputeTransform, isImputeSequence} from '../../transform';
 import {duplicate} from '../../util';
-import {VgImputeTransform, VgSignalRef} from '../../vega.schema';
+import {VgFormulaTransform, VgImputeTransform, VgSignalRef, VgWindowTransform} from '../../vega.schema';
+import {UnitModel} from '../unit';
 import {DataFlowNode} from './dataflow';
 
+// TODO Rename to ImputeNode
```

Comment:
Please add another commit to just rename it in this PR so we won't forget to do so.